js/private/test/image/structure/BUILD.bazel

```diff
@@ -26,6 +26,8 @@ js_image_layer(
     compression = "none",
     platform = ":amd64_linux",
     root = "/app",
+    # TODO: figure out why the digests_generated.sum is different on buildbuddy executors
```

Comment:
this is classic RBE; it has no access to the VCS and therefore has no knowledge of the `modes and bits` for the source files.

Reply:
and different RBE implementations assign different `permission` bits to the files.

Reply:
okay, there was already a no-remote-exec on the test below, but I think that never would have worked there because it's not the execution of `diff` that fails, it's the action producing the inputs to `diff`...